src/encoded/static/scss/encoded/modules/_static-pages.scss

```diff
@@ -551,44 +554,41 @@ $carousel-height : 240px;
 			&:hover, &:active {
 				text-decoration: none;
 			}
-			> .inner-container {
+			.inner-container {
 				position: relative;
 				min-height: $carousel-height - 10px;
 				background: #fff;
 				display: flex;
 				flex-direction: column;
-				//box-shadow: 0 1px 4px -1px rgba(0,0,0,0.5);
 				box-shadow: 0 1px 7px -2px rgba(0,0,0,0.5);
 				border-radius: 2px;
 				transition: box-shadow .5s;
 				background-color: #fff;
 				justify-content: center;
-				> .bg-image {
+				.bg-image {
 					background-color: #fff;
+					background-repeat: no-repeat;
 					opacity: 1;
 					left: 0;
 					top: 0;
 					bottom: 65px; //0;
 					right: 0;
-					background-size: cover;
-					background-position: 50%;
+					// background-size: cover;
+					background-position-x: 50%;
```

Comment:
Revert -- we want the background image to "cover" the tile, not be 100% size.

Consider that page (and transitively slide) width depends on device width, and some people might be looking at content from tablet, phone, or desktop. We want to slides to still show the same content at tablet size as they do on desktop size, and not cut up the image.

Reply:
@alexkb0009 , Actually, "cover" is not removed at all. There is a line in PageCarousel.js, that conditionally sets background-size depending on props:

`<div className="bg-image" style={img ? { 'backgroundImage': 'url(' + img + ')', 'backgroundSize': adjustImageHeight ? 'auto 100%' : 'cover' } : null} />`